modules/triggers-legacy-attribute-filters.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * /serverless/eventing/triggers/advanced-trigger-filters.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="triggers-legacy-attribute-filters_{context}"]
+= Legacy attributes filter
+
+The legacy attributes filter enables exact match filtering on any number of CloudEvents attributes, including extensions. Its functionality mirrors the exact filter dialect, and it is encouraged to transition to the exact filter whenever possible. However, for backward compatibility, the attributes filter remains available.
```

Comment:
To avoid passive tense, change to: "... and you are encouraged to transition..."

Reply:
done